build-logic-settings/settings.gradle.kts

```diff
@@ -49,24 +49,47 @@ dependencyResolutionManagement {
 // Duplicated settings because they are not inherited from root project
 // as described in https://docs.gradle.org/current/userguide/build_cache.html#sec:build_cache_composite
 // https://github.com/gradle/gradle/issues/18511
-@Suppress("UnstableApiUsage")
-val avitoGithubRemoteCacheHost: Provider<String> = settings.providers
-    .environmentVariable("GRADLE_CACHE_NODE_HOST")
-    .forUseAtConfigurationTime()
-
-val avitoGithubRemoteCachePush: String =
-    extra.properties.getOrDefault("avitoGithub.gradle.buildCache.remote.push", "false").toString()
+val useEnterprise = booleanProperty("avito.gradle.enterprise.enabled", false)
 
 buildCache {
-    remote<HttpBuildCache> {
-        setUrl("http://${avitoGithubRemoteCacheHost.orNull}/cache/")
-        isEnabled = avitoGithubRemoteCacheHost.orNull != null
-        isPush = avitoGithubRemoteCachePush.toBoolean()
-        isAllowUntrustedServer = true
-        isAllowInsecureProtocol = true
+    local {
+        isEnabled = true
```

Comment:
Even in docker? Have we measured it?

Reply:
You're right, we don't need in container